packages/core/src/Stores/client-store.js

```diff
@@ -1477,10 +1472,6 @@ export default class ClientStore extends BaseStore {
 
         const authorize_response = await this.setUserLogin(login_new_user);
 
-        if (action_param === 'signup') {
-            this.root_store.ui.setIsNewAccount();
-        }
-
         if (search) {
```

Comment:
shouldnt we delete below line
![Screenshot 2024-06-24 at 2 20 29 PM](https://github.com/binary-com/deriv-app/assets/84661147/b38f5390-93ab-4168-90a5-2280bfb73e51)

Reply:
yes will remove it 🙏